build_projects/dotnet-cli-build/DotNetRestoreProjectJson.cs

```diff
@@ -3,29 +3,29 @@
 
 namespace Microsoft.DotNet.Cli.Build
 {
-    public class DotNetRestore3 : DotNetTool
+    public class DotNetRestoreProjectJson : DotNetTool
     {
         protected override string Command
         {
-            get { return "restore3"; }
+            get { return "restore-projectjson"; }
```

Comment:
Instead of a 'dotnet restore-projectjson' command, can we get away with using 'dotnet nuget restore'?  That should be the equivalent right?  Then we don't need this temporary command.